e2e-tests/src/tests/cm-tabbar-toolbar.spec.ts

```diff
@@ -20,72 +20,6 @@ test.describe('CrossModel TabBar Toolbar', () => {
       await explorer.tabBarToolbar.waitForVisible();
    });
 
-   test('create new entity from tabbar toolbar', async () => {
```

Comment:
These two tests were redundant, as the same behavior of creation from tabbar is tested separately in the entity and relationship test suites.